docs/graphql/manual/api-reference/schema-metadata-api/computed-field.rst

```diff
@@ -22,8 +22,8 @@ add_computed_field
 There cannot be an existing column or relationship or computed field with
 the same name.
 
-Create a ``computed field`` called ``get_articles`` on an ``author`` *table*, using
-an SQL function called ``fetch_articles``:
+Create a ``computed field`` called ``full_name`` on an ``author`` *table*, using
```

Comment:
I was wondering if it would be good to also have the actual SQL function definition here, like in the schema part. It might be easier to visualize. What do you think?

Reply:
Uptill now we have been following the practice of not going into details of the examples we use in the API reference section. The API reference currently just shows API signatures but not the morivation behind the feature or the examples.

But it's a fair point Marion. Let's discuss this outside the domain of this PR as it actually impacts the entire API reference section.

Reply:
Thank you for the explanation, Rikin. Let's discuss it in our next call 🙂